scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -2,82 +2,12 @@
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class BuildVersion
+    public class BuildVersion : Version
     {
-        public int Major { get; set; }
-        public int Minor { get; set; }
-        public int Patch { get; set; }
-        public int CommitCount { get; set; }
-        public string CommitCountString => CommitCount.ToString("000000");
-        public string ReleaseSuffix { get; set; }
-
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
-        public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
+        public string VersionSuffix => $"{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
Also @brthor made changes to our [package targets](https://github.com/dotnet/cli/pull/2877/files#diff-cf3b35d6a17182b432693f932941e59cR217) to use the commit count as the version suffix for the nuget packages. But this fix was not made for the pack command in the [test targets](https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/TestTargets.cs#L165).